### DISCUSSION TITLE
Customise SCSS when installed as a hugo mod

### DISCUSSION THREAD

**Discussion by nickperkins** [Lednerb/bilberry-hugo-theme#531] [Q&A] [answered]
Love the theme.

I want to make some customisations to the scss files. The README suggests making these changes in the `/themes/bilberry-hugo-theme` folder, howeverI have installed the theme as a hugo mod. 

What would be the correct way of customising the scss in this instance?

**Comment by igor-baiborodine** [comment, marked as answer]
@nickperkins, currently, I don't think there is an easy way to customize colors when the theme is used as a Hugo module since the final CSS theme file is generated using `npm`. This issue will be resolved with the upcoming [v4 version](https://github.com/Lednerb/bilberry-hugo-theme/projects/2). A temporary workaround could be a your theme's fork with customized colors that is used as a Hugo module.

**Reply by jromal** [reply]
To fork the theme for just changing the colors seems like killing a mosquito with a bazooka. 
In the version v3, using NPM did the job. With v4 I have not managed. 

You said the issue would be resolved with v4 version. But the instructions on the readme are not clear to me and it is not said if they work when installed as a module or not.

I reversed to old v3 until I manage to change the colors. Any help is appreciated. Thanks.

**Reply by igor-baiborodine** [reply]
@jromal, here is an example that would work for all installation options in `v4`:

1. Select the color you want to customize:
![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/1027701/a27b5638-369b-4e66-8908-2d59f57854d9)

2. Override the value for the corresponding param `baseColor`  in the `config.toml` file: 
![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/1027701/f3f898eb-9c03-473f-b6a1-61498dd7bb16)

3. Test in local dev:
![image](https://github.com/Lednerb/bilberry-hugo-theme/assets/1027701/5168bebe-eb3f-46af-ade0-23412f7af6f3)

**Comment by igor-baiborodine** [comment]
... or, as another option, in the `https://github.com/nickperkins/nickperkins.au/tree/main/_vendor/github.com/Lednerb/bilberry-hugo-theme/v3` folder, try customizing colors and executing `npm run production`. Then commit and push the changes. I think it should work.
